scalafix-core/src/main/scala/scalafix/package.scala

```diff
@@ -26,8 +26,8 @@ package object scalafix {
       extends SyntacticPatchOps(ctx) {
     def semanticOps: SyntacticPatchOps = this
   }
-  implicit class XtensionSeqPatch(patches: Seq[Patch]) {
-    def asPatch: Patch = Patch.fromSeq(patches)
+  implicit class XtensionSeqPatch(patches: Iterable[Patch]) {
+    def asPatch: Patch = Patch.fromIterable(patches)
```

Comment:
What's the motivation behind `Iterable`? I wonder what use cases you found for this more general type.